src/Discovery/Discovery.tsx

```diff
@@ -279,17 +288,20 @@ const Discovery: React.FunctionComponent<Props> = (props: Props) => {
     const searchableFields = config.features.search.searchBar.searchableTextFields;
     if (searchableFields) {
       searchableFields.forEach((field) => {
-        search.addIndex(field);
+        const formattedFields = formatSearchIndex(field);
+        formattedFields.forEach((formattedField) => search.addIndex(formattedField));
```

Comment:
```suggestion
        search.addIndex(formattedFields);
```
During my testing, this works, but the way of adding each individual field into `search` doesn't

Reply:
Suggested applied! Thank you.